packages/vite/src/node/plugins/assetImportMetaUrl.ts

```diff
@@ -21,6 +18,8 @@ export function assetImportMetaUrlPlugin(config: ResolvedConfig): Plugin {
   return {
     name: 'asset-import-meta-url',
     async transform(code, id, ssr) {
+      const importMetaUrlRE =
```

Comment:
Should we move it further down? So it will only be created if the condition is fulfilled.

Reply:
updated